docs/node-types.md

```diff
@@ -0,0 +1,53 @@
+# Node Types in IPFS Companion
+
+![screenshot of node type toggle](https://user-images.githubusercontent.com/157609/39421672-59010924-4c6a-11e8-9e64-6b5d5f5f2768.png)
+
+> **TL;DR** When in doubt, run go-ipfs as External node
```

Comment:
I agree with the tl;dr. Add a link to https://ipfs.io/docs/getting-started/ so they have all they need!